reversinglabs_tiscalev2_connector.py

```diff
@@ -0,0 +1,198 @@
+# File: reversinglabs_tiscalev2_connector.py
+#
+# Copyright (c) ReversingLabs, 2023
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# !/usr/bin/python
```

Comment:
Can probably remove this boilerplate code

Reply:
Done